h5py/tests/test_group.py

```diff
@@ -1038,3 +1038,30 @@ def test_validity(self):
         Group.__delitem__
         Group.__iter__
         Group.__len__
+
+class TestRequire(TestCase):
```

Comment:
There's already a `TestRequire` class in the file above. Reusing the name means that doesn't run, which is why we're seeing a coverage drop. Can you add the test methods to that?